shell/browser/net/system_network_context_manager.cc

```diff
@@ -195,6 +195,7 @@ SystemNetworkContextManager::CreateDefaultNetworkContextParams() {
 void SystemNetworkContextManager::ConfigureDefaultNetworkContextParams(
     network::mojom::NetworkContextParams* network_context_params) {
   network_context_params->enable_brotli = true;
+  network_context_params->enable_zstd = true;
```

Comment:
[Chrome is checking for a feature flag](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/net/system_network_context_manager.cc;l=844-846;drc=c1b9831c8c232ab9470645977a18527cfa7bb993), can we also do the same?
```suggestion
  network_context_params->enable_zstd = 
      base::FeatureList::IsEnabled(net::features::kZstdContentEncoding);
```

If for some reason this encoding breaks apps downstream, this provides an easy escape hatch to disable it.

Reply:
Sounds great, thanks for the suggestion!

Reply:
Might need to add `#include "net/base/features.h"` btw, but let's see if the build passes anyway 😄